gradle/performanceTest.gradle

```diff
@@ -43,9 +43,13 @@ dependencies {
     //so that implicit help tasks are available:
     performanceTestRuntime project(':diagnostics')
 
-    //So that the wrapper and init task are added when performanceTests are run via commandline
-    //above can be removed when we implement the auto-apply plugins
     performanceTestRuntime project(':buildInit')
+    performanceTestRuntime project(':buildCacheHttp')
+    performanceTestRuntime project(':codeQuality')
+    performanceTestRuntime project(':ide')
+    performanceTestRuntime project(':maven')
+    performanceTestRuntime project(':testingNative')
+    performanceTestRuntime project(':toolingApiBuilders')
```

Comment:
We could add more dependencies here to have a _full_ distribution for performance tests. Currently the following jars are not in the distribution:

![missing in perf](https://user-images.githubusercontent.com/1963746/30701119-c337f5f2-9ee9-11e7-90e5-61a5ce095923.png)